ror-tools/src/test/scala/tech/beshu/ror/tools/RorToolsAppSuite.scala

```diff
@@ -81,7 +84,29 @@ class RorToolsAppSuite extends AnyWordSpec with BeforeAndAfterAll with BeforeAnd
           |""".stripMargin
       )
     }
-    "Patching does not start when user declines to accept implications of patching (in arg)" in {
+    "Patching successful first time, on second try not started because already patched" in {
```

Comment:
I think that the tests now cover all cases from diagrams https://github.com/sscarduzio/readonlyrest_kbn/pull/581

Reply:
did you check the [ROR KBN patcher tests](https://github.com/sscarduzio/readonlyrest_kbn/blob/develop/test/KibanaPatcher.test.ts)?